lib/logidze.rb

```diff
@@ -7,17 +7,22 @@
 module Logidze
   require "logidze/history"
   require "logidze/model"
+  require "logidze/model/time_helper"
+  require "logidze/model/active_record"
+  require "logidze/model/sequel"
   require "logidze/versioned_association"
   require "logidze/ignore_log_data"
   require "logidze/has_logidze"
-  require "logidze/meta"
-
-  extend Logidze::Meta
+  require "logidze/adapter/base"
```

Comment:
The "adapter" name is too generic; what about "database_adapter"? Or "connection_adapters" like in Active Record.

Reply:
Let's stick with the Rails naming.